README.md

```diff
@@ -1,59 +1,53 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Leaderboard
 
-# Project Name
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> One paragraph statement about the project.
+# Leaderboard
 
-![screenshot](./app_screenshot.png)
+![screenshot](./capture.PNG)
 
 Additional description about the project and its features.
```

Comment:
- [ ] Kindly change this and give a description of what is your project about or what you are trying to accomplish with it 😄 
    - A description in your README file will save time for other developers and will help recruiters to go through it 🚀